src/main/java/it/gov/pagopa/payment/service/qrcode/QRCodeAuthPaymentServiceImpl.java

```diff
@@ -18,14 +20,16 @@ public class QRCodeAuthPaymentServiceImpl implements QRCodeAuthPaymentService {
   private final TransactionInProgressRepository transactionInProgressRepository;
   private final RewardCalculatorConnector rewardCalculatorConnector;
   private final AuthPaymentMapper requestMapper;
+  private final AuthorizationNotificationProducer authorizationNotificationProducer;
 
   public QRCodeAuthPaymentServiceImpl(
-      TransactionInProgressRepository transactionInProgressRepository,
-      RewardCalculatorConnector rewardCalculatorConnector,
-      AuthPaymentMapper requestMapper) {
+          TransactionInProgressRepository transactionInProgressRepository,
+          RewardCalculatorConnector rewardCalculatorConnector,
+          AuthPaymentMapper requestMapper, AuthorizationNotificationProducer authorizationNotificationProducer) {
```

Comment:
let's keep the same format
```suggestion
          AuthPaymentMapper requestMapper,
          AuthorizationNotificationProducer authorizationNotificationProducer) {
```